requirements.txt

```diff
@@ -17,5 +17,5 @@ pyproj==1.9.3
 prettytable==0.7.2
 # When updating nupic.bindings, also update any shared dependencies to keep
 # versions in sync.
-nupic.bindings==0.4.6
+nupic.bindings>=0.4.6
```

Comment:
I don't really think that's a big deal. This will affect developers (not users), and I think we can provide instructions on how to help developers recover. As developers, how else are we going to test nupic.core versions against nupic unless it allows it?